Discord Mafia Bot .Net/Core/GameManager.cs

```diff
@@ -508,6 +508,8 @@ private static List<MafiaRole> HandleSetup(GamePlayerList g)
             builder.AddField(winField);
             builder.AddField(loseField);
             await (game.GameChat as IMessageChannel).SendMessageAsync("", false, builder.Build());
+            await (IGuildChannel.signupchannel as IMessageChannel).SendMessageAsync("", false, builder.Build());
```

Comment:
you want to save `signup*C*hannel` inside of `game` that way you can call `game.signupChannel` 🙂